src/OrbitGl/ScopeTree.h

```diff
@@ -85,6 +86,8 @@ class ScopeTree {
   GetOrderedNodesByDepth() const {
     return ordered_nodes_by_depth_;
   }
+  [[nodiscard]] ScopeT* GetRight(uint32_t depth, uint64_t time) const;
+  [[nodiscard]] ScopeT* GetLeft(uint32_t depth, uint64_t time) const;
```

Comment:
nit. maybe `FindFirstScopeAtDepthAfterTime` and `FindFirstScopeAtDepthBeforeTime` would be more descriptive?

Reply:
Also can we return `const ScopeT*`?

Reply:
Changed to const. After refactoring, I named them FindNextScopeAtDepth and FindPreviousScopeAtDepth.